packages/browsers/src/CLI.ts

```diff
@@ -50,33 +50,86 @@ export class CLI {
+          buildId: this.#parseBuildId(opt),
+        };
+      },
+    });
+  }
+
+  #definePlatformParameter(yargs: yargs.Argv<unknown>) {
+    yargs.option('platform', {
+      type: 'string',
+      desc: 'Platform that the binary needs to be compatible with.',
+      choices: Object.values(BrowserPlatform),
+      defaultDescription: 'Auto-detected by default.',
+    });
+  }
+
+  #definePathParameter(yargs: yargs.Argv<unknown>) {
+    yargs.option('path', {
+      type: 'string',
+      desc: 'Path to the root folder for the browser downloads and installation',
+      default: process.cwd(),
+    });
+  }
+
   async run(argv: string[]): Promise<void> {
     await yargs(hideBin(argv))
       .command(
         'install <browser>',
         'Download and install the specified browser',
         yargs => {
-          yargs.positional('browser', {
-            description: 'The browser version',
-            type: 'string',
-            coerce: (opt): InstallArgs['browser'] => {
-              return {
-                name: this.#parseBrowser(opt),
-                buildId: this.#parseBuildId(opt),
-              };
-            },
-          });
-          yargs.option('platform', {
-            type: 'string',
-            desc: 'Platform that the binary needs to be compatible with.',
-            choices: Object.values(BrowserPlatform),
-            defaultDescription: 'Auto-detected by default.',
-          });
-          yargs.option('path', {
-            type: 'string',
-            desc: 'Path where the browsers will be downloaded to and installed from',
-            default: process.cwd(),
+          this.#defineBrowserParameter(yargs);
+          this.#definePlatformParameter(yargs);
+          this.#definePathParameter(yargs);
+        },
+        async argv => {
+          const args = argv as unknown as InstallArgs;
+          args.platform ??= detectBrowserPlatform();
+          if (!args.platform) {
+            throw new Error(`Could not resolve the current platform`);
+          }
+          args.browser.buildId = await resolveBuildId(
+            args.browser.name,
+            args.platform,
+            args.browser.buildId
+          );
+          await fetch({
+            browser: args.browser.name,
+            buildId: args.browser.buildId,
+            platform: args.platform,
+            cacheDir: args.path ?? this.#cachePath,
+            downloadProgressCallback: this.#makeProgressCallback(
+              args.browser.name,
+              args.browser.buildId
+            ),
           });
+          console.log(
+            `${args.browser.name}@${
+              args.browser.buildId
+            } ${computeExecutablePath({
+              browser: args.browser.name,
+              buildId: args.browser.buildId,
+              cacheDir: args.path ?? this.#cachePath,
+              platform: args.platform,
+            })}`
+          );
+        }
+      )
+      .command(
+        'downloads <browser>',
```

Comment:
Is the idea for this to be an alias to `install`, the code looks the same? Yargs supports that -> https://github.com/yargs/yargs/blob/main/docs/advanced.md#command-aliases

Reply:
I have forgotten to remove the command I have been trying out locally.